### DISCUSSION TITLE
AGS overview on Gentoo

### DISCUSSION THREAD

**Discussion by JaKooLit** [JaKooLit/Hyprland-Dots#286] [Q&A] [answered]
Would like to ask here first before I reformat my system and move back to multilib (with 32bit) support profile

https://github.com/Aylur/ags/issues/419

@SherLock707 @JohnRTitor 

Any ideas? I asked Aylur but seems he dont know as well

I have asked Gentoo Forums but nobody seems to use use AGS overview there

On my desktop with Multilib, it works.

But both on my mini-PC and Laptop has a no-multilib profile and I am experiencing this to both system

ps. Multilib is with 32bit support
NO-Multilib means only 64bit

**Comment by SherLock707** [comment]
No idea what mutilib profile is, will look into it and how its affecting ags overview..

**Reply by JaKooLit** [reply]
Ive edited my post

Multilib is with 32bit support
NO-Multilib means only 64bit

**Comment by JaKooLit** [comment, marked as answer]
> No idea what mutilib profile is, will look into it and how its affecting ags overview..

Found the culprit.. apparently, the ags code you've shared relies on xrandr

**Reply by SherLock707** [reply]
Yes it does... Should we swap out to something else?

**Reply by JaKooLit** [reply]
if we can omit the xrandr better

Ive tried deleting that line but it seems it does something..

if we can utilize the "hyprctl monitors" instead thats better

**Reply by SherLock707** [reply]
Okay, I'll try to make that PR.
